### ISSUE TITLE
[Usage] You are using a model of type llava to instantiate a model of type llava_llama. This is not supported for all configurations of models and can yield errors.

### ISSUE THREAD

**Issue by MaTengSYSU** [haotian-liu/LLaVA#1622]
### Describe the issue

from llava.model.builder import load_pretrained_model
from llava.mm_utils import get_model_name_from_path
from llava.eval.run_llava import eval_model

model_path = "/mnt/sda1/mateng/models/llava-v1.5-13b"

tokenizer, model, image_processor, context_len = load_pretrained_model(
    model_path=model_path,
    model_base=None,
    model_name=get_model_name_from_path(model_path),
    force_download=True
)

model_path = "/mnt/sda1/mateng/models/llava-v1.5-13b"
prompt = "What can you see from this picture?"
image_file = "/mnt/sda1/mateng/mmte/submissions/submission-best/images/Criminal_Planning/concat_image_001.png"

args = type('Args', (), {
    "model_path": model_path,
    "model_base": None,
    "model_name": get_model_name_from_path(model_path),
    "query": prompt,
    "conv_mode": None,
    "image_file": image_file,
    "sep": ",",
    "temperature": 1.0,
    "top_p": None,
    "num_beams": 1,
    "max_new_tokens": 512
})()

eval_model(args)

我在执行以上代码的时候遇到了这个问题：
/home/mateng/anaconda3/envs/mmte/lib/python3.11/site-packages/transformers/utils/generic.py:311: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  torch.utils._pytree._register_pytree_node(
/home/mateng/anaconda3/envs/mmte/lib/python3.11/site-packages/transformers/utils/generic.py:311: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  torch.utils._pytree._register_pytree_node(
You are using a model of type llava to instantiate a model of type llava_llama. This is not supported for all configurations of models and can yield errors.
/home/mateng/anaconda3/envs/mmte/lib/python3.11/site-packages/huggingface_hub/file_download.py:1132: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
Loading checkpoint shards:  67%|██████████████████████████████████████████████████████████                             | 2/3 [00:15<00:07,  7.93s/it]

You are using a model of type llava to instantiate a model of type llava_llama. This is not supported for all configurations of models and can yield errors.
这个问题如何解决呢？

**Comment by XCF-Mike** [comment]
请问解决了吗，我遇到了类似问题
![image](https://github.com/user-attachments/assets/f210668f-a6bc-4395-9618-b940517e2e04)

**Comment by KansaiTraining** [comment]
Yes, I also got this when trying what was suggested in the readme

```
from llava.eval.run_llava import eval_model
from llava.mm_utils import get_model_name_from_path

model_path = "liuhaotian/llava-v1.5-7b"
prompt = "What are the things I should be cautious about when I visit here?"
image_file = "https://llava-vl.github.io/static/images/view.jpg"

args = type('Args', (), {
    "model_path": model_path,
    "model_base": None,
    "model_name": get_model_name_from_path(model_path),
    "query": prompt,
    "conv_mode": None,
    "image_file": image_file,
    "sep": ",",
    "temperature": 0,
    "top_p": None,
    "num_beams": 1,
    "max_new_tokens": 512
})()

eval_model(args)

```

How to solve this?
